lib/s3.py

```diff
@@ -2,8 +2,17 @@
 
 from abc import ABCMeta, abstractmethod
 from io import BytesIO
-from boto.s3.connection import S3Connection
-from boto.s3.key import Key
+import sys
```

Comment:
minor issues if you want to address them: we prefer to import only those things we're using (so not all of boto3, not all of sys)